evm-adapters/src/call_tracing.rs

```diff
@@ -575,3 +620,56 @@ fn diff_score(bytecode1: &[u8], bytecode2: &[u8]) -> f64 {
     // println!("diff_score {}", diff_chars as f64 / cutoff_len as f64);
     diff_chars as f64 / cutoff_len as f64
 }
+
+fn decode_solidity_reverts(data: &[u8]) -> Option<&str> {
+    if data.len() >= 4 {
```

Comment:
There is also https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.starts_with

so this can be simplified a bit

```rust
if data.starts_with(&[78, 72, 123, 113]) {
   return None;
}

let msg = match data.last()? {
    1 => "Assertion violated",
    //--snip
   _ => return None
}

Some(msg)



```